web/src/services/Api.ts

```diff
@@ -58,7 +58,7 @@ export function toData<T>(resp: AxiosResponse<ServiceResponse<T>>): T | undefine
 export function hasServiceError<T>(resp: AxiosResponse<ServiceResponse<T>>) {
     const errResp = toErrorResponse(resp);
     if (errResp && errResp.status === "KO") {
-        return true;
+        return [true, errResp.message];
```

Comment:
I advise you to use a dict instead of a tuple, that way at the caller you can refer to a name instead of [0] which is more user friendly for other developers.